splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/SelectTimeTravelIT.java

```diff
@@ -0,0 +1,253 @@
+package com.splicemachine.derby.impl.sql.execute.operations;
+
+import com.splicemachine.derby.test.framework.*;
+import org.junit.*;
+import org.junit.rules.RuleChain;
+import org.junit.rules.TestRule;
+import org.junit.runner.Description;
+
+import java.sql.*;
+
+public class SelectTimeTravelIT {
+
+    private static final String SCHEMA = SelectTimeTravelIT.class.getSimpleName().toUpperCase();
+
+    @ClassRule
+    public static SpliceWatcher watcher = new SpliceWatcher(SCHEMA);
+
+    @ClassRule
+    public static SpliceSchemaWatcher spliceSchemaWatcher = new SpliceSchemaWatcher(SCHEMA);
+
+    private long GetTxId() throws SQLException {
+        long result = -1;
+        ResultSet rs =  watcher.executeQuery("CALL SYSCS_UTIL.SYSCS_GET_CURRENT_TRANSACTION()");
+        Assert.assertTrue(rs.next());
+        result = rs.getLong(1);
+        Assert.assertFalse(rs.next());
+        return result;
+    }
+
+    static int counter = 0;
+    private static String GenerateTableName() {
+        return "T" + counter++;
+    }
+
+    @Test
+    public void testTimeTravelWorks() throws Exception {
+        String tbl = GenerateTableName();
```

Comment:
do we really need this? maybe sth like "testTimeTravelWorksT" would be more intuitive? the SpliceWatcher will drop these tables automatically, so you can run the test multiple times.